config/settings/common.py

```diff
@@ -1,5 +1,8 @@
 """
-:copyright (c) 2014 - 2017, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:copyright (c) 2014 - 2016, The Regents of the University of California,
```

Comment:
can you fix the header in this file? make sure that the dates are 2014-2017 and fix the line feeds.